src/Stripe.net/Services/SubscriptionItems/SubscriptionItemService.cs

```diff
@@ -6,7 +6,7 @@ namespace Stripe
 
     public class SubscriptionItemService : Service<SubscriptionItem>,
         ICreatable<SubscriptionItem, SubscriptionItemCreateOptions>,
-        IDeletable<SubscriptionItem>,
+        IDeletableWithParams<SubscriptionItem, SubscriptionItemDeleteOptions>,
```

Comment:
This seemed the least intrusive. The alternative is to change all `Delete` to take params by default even though this is the only method today that supports this (and Terminal objects for legacy reasons that is unsupported anyway).

What do you think is best here?

Reply:
I'm not sure there's a lot of value in adding a different interface for just this one service. I think we should just remove the `IDeletable` interface from the service, and in a future major version update the interface to accept params (which means creating parameter classes for all delete methods), like we did for the `IRetrievable` interface a while back.

Reply:
Fair. Let's ignore that part and answer the real question of: would we move all delete methods to take an empty options hash for that one exception?

Reply:
I'm personally in favor. The user impact is limited (just need to update callsites to `Delete(id, requestOptions)` to be either `Delete(id, null, requestOptions)` or `Delete(id, requestOptions: requestOptions)`) and it makes the API more consistent and future proof.